examples/nidcpower/measure-record.py

```diff
@@ -204,18 +203,15 @@ def _on_close(event):
 
 except grpc.RpcError as rpc_error:
     error_message = rpc_error.details()
+    for key, value in rpc_error.trailing_metadata() or []:
+        if key == "ni-error":
+            error_message += f"\nError status: {value}"
```

Comment:
Does this value include the error description or message or are we just printing out the code now?